src/trans/internal/sugaw_mod.F90

```diff
@@ -108,6 +107,7 @@ SUBROUTINE SUGAW(KDGL,KM,KN,PL,PW,PANM,PFN)
 
 ! computations in extended precision for alternative root finding
 ! which also works for associated polynomials (m>0)
+INTEGER, PARAMETER :: JPRH = JPRD
```

Comment:
This one I need to think about a bit. It looks like we are removing [the option to compute the roots in extended precision](https://github.com/ecmwf-ifs/fiat/blob/main/src/parkind/parkind2.F90#L26). If we don't need this anymore, and double precision is fine, better to be transparent and just replace all instances of JPRH with JPRD.

Reply:
Yes I don't think we need JPRH. At least parkind1 has not recently compiled JPRH differently.
If agreed by others as well we can follow your proposal.

Reply:
I can just create an issue so we deal with this in a future PR.

Reply:
Thanks @samhatfield for creating #62 for follow-up.